bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameter.java

```diff
@@ -81,6 +83,8 @@
     private String unitLabel;
 
     private String context;
+
+    @SerializedName("default")
```

Comment:
I second @maggu2810 suggestion. On the other hand we write ConfigurationParameter data to the JSONStorage which is done via the model classes. Should all import/export of JSON be done using DTOs? If we want to keep the JSONStorage in sync with the REST resources this would make sense. This would also mean to map to/from DTOs on all external borders and only use the model class internally.

@kaikreuzer @maggu2810 @sjka wdyt?